scripts/build-scies.sh

```diff
@@ -1,4 +1,6 @@
 #! /bin/bash
```

Comment:
please remove space.

Reply:
All the other scripts in our repo look like this.